prisma/schema.prisma

```diff
@@ -292,10 +293,10 @@ enum WebhookTriggerEvents {
 }
 
 model Webhook {
-  id            String      @unique @id
+  id            String                 @id @unique
```

Comment:
feels like we want to add foreign key on userId here. So when user is deleted web hooks should be deleted too. Can create another ticket. What do you think?

Reply:
Yeah, I've seen similar things in a few places - I don't want to touch anything else than strictly needed for Prisma 3 upgrade in this PR, so let's do that as follow-ups.

Another thing that would be nice would be to have consistent table naming & get rid of all those `@@map`